### PR TITLE
Tomporvaz/add site button

### LINKED ISSUE #323: Update the wording on the Contribute Button on Desktop
**Is your feature request related to a problem? Please describe.**
Users can sometimes be confused around the purpose of the Contribute text and what the button does (Contribute may sometimes imply donations). In response, we would like to change the Contribute button text to "Add Site"

This would require an update to the Toolbar component.

**Describe the solution you'd like**
We would like the button to match the visual in this prototype: https://www.figma.com/proto/VGGqwl3Eq3GQIAO9I6LoNW/PHLASK?page-id=4510%3A29556&type=design&node-id=8105-140052&viewport=889%2C-851%2C0.09&scaling=scale-down&starting-point-node-id=8105%3A139899&show-proto-sidebar=1&mode=design

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by tomporvaz**
# Pull Request 323

## Change Summary

Changed the Contribute button text to Add Site
Moved textTransform: none up to button sx

## Change Reason

Changed to avoid confusion about contributing information not money.

## Verification [Optional]
Phlask_2023-07-18 21-10-56

Related Issue: #323

### CODE REVIEW

File: src/components/Toolbar/Toolbar.js [hunk 1]
```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
...
-              flexDirection: 'column'
+              flexDirection: 'column',
+              textTransform: 'none'
             }}
           >
             <ResourceIcon />
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Resources
-            </Typography>
+            <Typography fontSize={'small'}>Resources</Typography>
```
Comment (RNR1) [inline src/components/Toolbar/Toolbar.js]: Curly braces are not necessary for the `fontSize` prop:
```suggestion
            <Typography fontSize="small">Resources</Typography>
```

File: src/components/Toolbar/Toolbar.js [hunk 2]
```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
...
-              flexDirection: 'column'
+              flexDirection: 'column',
+              textTransform: 'none'
             }}
           >
             <FilterIcon />
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Filter
-            </Typography>
+            <Typography fontSize={'small'}>Filter</Typography>
```
Comment (RNR1) [inline src/components/Toolbar/Toolbar.js]: Same here:
```suggestion
            <Typography fontSize="small">Filter</Typography>
```

File: src/components/Toolbar/Toolbar.js [hunk 3]
```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
...
-              flexDirection: 'column'
+              flexDirection: 'column',
+              textTransform: 'none'
             }}
           >
             <SearchIcon />
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Search
-            </Typography>
+            <Typography fontSize={'small'}>Search</Typography>
```
Comment (RNR1) [inline src/components/Toolbar/Toolbar.js]: Same here
```suggestion
            <Typography fontSize="small">Search</Typography>
```

File: src/components/Toolbar/Toolbar.js [hunk 4]
```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
...
               display: 'flex',
-              flexDirection: 'column'
+              flexDirection: 'column',
+              textTransform: 'none'
             }}
           >
             <ContributeIcon />
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Contribute
+            <Typography noWrap fontSize={'small'}>
```
Comment (RNR1) [inline src/components/Toolbar/Toolbar.js]: Same here:
```suggestion
            <Typography noWrap fontSize="small">
```

**Review by RNR1 — CHANGES_REQUESTED** [review]
Looking great overall, just a few styling changes suggested here

**Comment by tomporvaz** [comment]
@RNR1 I have removed the all of the curly brackets requested, and I also found more in the mobile code.  Searching for curly brackets, I realized that the toolbar is duplicated for mobile, so I updated the contribute button on mobile too.  I think this is complete, but let me know if there is anything else.

**Review by RNR1 — APPROVED** [review]
Thanks! Looking good

**Pull request merged**